config.toml

```diff
@@ -138,6 +138,18 @@ disableHugoGeneratorInject = true
     text = "Write HTTP handlers in WebAssembly with a minimal amount of work."
     url = "https://github.com/deislabs/wagi"
 
+    [[params.section1.tiles]]
```

Comment:
The listing of projects is currently in alphabetical order but this puts these at the end, unsorted. Can you move the sections into their properly place by alphabetical order?

Reply:
Sure!